.travis.yml

```diff
@@ -42,6 +42,10 @@ matrix:
     - php: 5.6
       env: DB=mysql
     - php: 5.6
-      env: DB=mysql;TC=lint
+      env: DB=mysql TC=lint
+    - php: 5.6
+      env: DB=mysql CORE_BRANCH=stable10
```

Comment:
since ``CORE_BRANCH=stable10`` already in ``global`` then this matrix entry will duplicate the one 2 entries above.

Reply:
thanks for noticing - fixed - THX